enterprise/frontend/src/embedding-sdk/components/private/data-picker/DataSelector.jsx

```diff
@@ -0,0 +1,1102 @@
+/* eslint-disable react/prop-types */
```

Comment:
Every new file is copied from `/Users/kelvin/workspace/metabase/frontend/src/metabase/query_builder/components/DataSelector/*` 1:1. File names and component names stay the same for the sake of reviewing. I might change them later on but for this PR, they better stay the same.

If there's any diff I'll comment inline. You can also check the diff between the 2 directories with
```sh
diff -burN enterprise/frontend/src/embedding-sdk/components/private/data-picker frontend/src/metabase/query_builder/components/DataSelector
```

Please note that there would be huge diff for SimpleDataPicker, but other than that they should relatively stay the same.